extensions/exchanges/sim/exchange.js

```diff
@@ -161,6 +161,12 @@ module.exports = function sim (conf, s) {
       }, latency)
     },
 
+    setFees: function(opts) {
+      if (so.mode === 'paper') {
```

Comment:
Maybe I'm missing something, but why would we only want to do this in paper? (And not sim?)

Reply:
Original implementation had it only in live/paper as the feature relies on a (real time) timer we cannot "sim", but then again it wouldn't hurt.

Reply:
Actually it does hurt :grin: 

Just did some sim runs with debugging enabled and noticed that setFees will get called on each new trade day causing us to hammer the exchange with requests into oblivion.

Reverted to only work in paper.